base/testfiles/tlb-latexrelease-rollback-003-often.luatex.tlg

```diff
@@ -278,9 +278,11 @@ Skipping: [....-..-..] Spaces in file names and hooks on input line ....
 Applying: [....-..-..] Spaces in file names and hooks on input line ....
 Skipping: [....-..-..] Quote file names on input line ....
 Applying: [....-..-..] Quote file names on input line ....
-Skipping: [....-..-..] manage unbalanced conditionals on input line ....
-Skipping: [....-..-..] manage unbalanced conditionals on input line ....
-Applying: [....-..-..] manage unbalanced conditionals on input line ....
+Skipping: [....-..-..] Cache file status on input line ....
+Skipping: [....-..-..] Cache file status on input line ....
+Applying: [....-..-..] Cache file status on input line ....
+Already applied: [....-..-..] manage unbalanced conditionals on input line ....
+Already applied: [....-..-..] manage unbalanced conditionals on input line ....
```

Comment:
No, not sure at all: I find rollback very tricky! I'll have a look again.

Reply:
Ah, I know: I didn't really know quite how to handle this. We have a rollback for the previous change to `\IfFileExists`, but the new patch builds on that. So I was not sure how we want to express that: ideas welcome.

Reply:
@FrankMittelbach  Sure, what I'm not clear on is when we have two changes to a piece of code, one of which comes after the other. I could simply use the same string for both, and it would 'work', but it would be misleading. I'll look for a similar example and try to get this right ...

Reply:
the label is what needs to be the same (and this is meant to be internal ie just a label like the cs that is modified, the text can be changed from one change to the next.

Reply:
Ok, I’ll look at the labels

Reply:
Better now using a single name?

Reply:
either or. You could have just told me that this is due to them all using the same label :-) I was only surprised to see the change. With a bit more care I could have figured out that all is well because it is the same lable with a different explanation.